tests/ChildRuleTest.php

```diff
@@ -126,19 +125,14 @@ public function testUnexpectedChild(): void {
   }
 
   public function testSingleChild(): void {
-    $elems = Vector {
-      <test:any-children />,
-      <test:single-child />,
-      <test:optional-child />,
-      <test:any-number-of-child />,
-      <test:at-least-one-child />,
-      <test:either-of-two-children />,
-      <test:nested-rule />,
-      <test:category-child />,
+    $elems = Vector { <test:any-children />, <test:single-child />,
```

Comment:
Is this what `hackfmt` did? Seems like we need to revisit that.

Reply:
Yes

Reply:
Seems like hackfmt is not idempotent with these cases